### ISSUE TITLE
last character of filename is truncated in SHO VAR/TREE

### ISSUE THREAD

**Issue by karlmsmith** [NOAA-PMEL/Ferret#1494]
**Reported by andrew wittenberg on 8 Dec 2014 15:38 UTC**
Note the missing "y" at the end of "coads_climatology" and "levitus_climatology".  Also happens for full pathnames -- the extension displays as ".n" instead of ".nc".

```
 	NOAA/PMEL TMAP
 	FERRET v6.93  
 	Linux 2.6.32-504.el6.x86_64 64-bit - 11/13/14
 	 8-Dec-14 10:31     

yes? use coads_climatology
yes? use levitus_climatology
yes? use monthly_navy_winds
yes? let a = sst
yes? let b = temp
yes? let c = a[let d = b[d=2](d=1]
yes?)
yes? let f = c+d
yes? sho var/tree f
   in default dataset monthly_navy_winds
   F = C+D
     C = A[      A[d=coads_climatolog](D=1]
) = SST
     D = B[      B[d=levitus_climatolog](D=2]
) = TEMP
```

Migrated-From: http://dunkel.pmel.noaa.gov/trac/ferret/ticket/2222

**Comment by karlmsmith** [comment]
**Modified by steven.c.hankin on 8 Dec 2014 18:19 UTC**

**Comment by karlmsmith** [comment]
**Comment by andrew.wittenberg on 14 Jun 2015 00:42 UTC**
Not sure when it happened, but this appears to be fixed in v6.95_rc1 (dated 6/12/2015).
